src/Core/Common/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ add_example( CreateAnother )
 add_example( CreateAFixedArray )
 add_example( AddOffsetToIndex )
 add_example( GetNameOfClass )
-add_example( CreateAnImageRegion )
+add_example( CreateAImageRegion )
```

Comment:
I think `an image` is correct. Is this a spelling change or to ease alphabetical sorting?